cpp/cmake_modules/FindOpenSSLAlt.cmake

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+find_package(OpenSSL)
+
+if (OPENSSL_FOUND)
+    if (OPENSSL_VERSION LESS "1.1.0")
+        message(SEND_ERROR "The OpenSSL must be grater than 1.0.0, the actual version is ${OPENSSL_VERSION}")
```

Comment:
>> grater than 1.0.0

greater than 1.1.0

Reply:
I meant greater than "1.1.0" (or rather greater than or equal to 1.1.0 to be correct)

Reply:
I changed now, check if it is correct